README.md

```diff
@@ -142,3 +142,16 @@ test lang_tests::unused_var ... ok
 
 test result: ok. 2 passed; 0 failed; 0 ignored; 0 measured; 2 filtered out
 ```
+
+## Integration with Cargo.
+
+Tests created with lang_tester can be used as part of an existing test suite.
+The `Cargo.toml` file can be modified so that tests created with lang_tester
+will run with the `cargo test` command.
```

Comment:
Can we put something like "For example, if the Rust source file that runs your lang tests is `lang_tests/run.rs` then add the following to your `Cargo.toml`"? Maybe not that exactly, but hopefully you get the rough idea!

Reply:
Sorry, I forgot to comment saying I had changed this :man_facepalming: What do you think now?